tests/import_test.go

```diff
@@ -454,7 +454,7 @@ var _ = Describe("[rfe_id:1118][crit:high][vendor:cnv-qe@redhat.com][level:compo
 		afterCMD(portForwardCmd)
 	})
 
-	It("[test_id:4970]Import pod should have prometheus stats available while importing", func() {
+	PIt("[test_id:4970]Import pod should have prometheus stats available while importing", func() {
```

Comment:
Right, we are skipping it because the prometheus endpoint isn't started at all if we cannot get the size from the endpoint. Before when we used nbdkit it would provide us progress, but now that we are downloading directly from the endpoint, we need the size to calculate the progress. And since our endpoint doesn't provide us this information the test fails to connect to the prometheus endpoint

Reply:
I hate to re-run all the tests, just with adding a comment. But if you insist I can do it.